internal/services/policy/policy_set_definition_resource.go

```diff
@@ -272,21 +214,13 @@ func resourceArmPolicySetDefinitionCreate(d *pluginsdk.ResourceData, meta interf
 
 	name := d.Get("name").(string)
 	managementGroupName := ""
-	if !features.ThreePointOhBeta() {
-		if v, ok := d.GetOk("management_group_name"); ok {
-			managementGroupName = v.(string)
-		}
-	}
-
-	if v, ok := d.GetOk("management_group_id"); ok {
-		id, err := mgmtGrpParse.ManagementGroupID(v.(string))
-		if err != nil {
-			return err
-		}
-		managementGroupName = id.Name
+	managementGroupID, err := mgmtGrpParse.ManagementGroupID(d.Get("management_group_id").(string))
```

Comment:
looks like the remove the existence check of "management_group_id" cause the issue #16942